### PR TITLE
Sarahwood registration content updates

### PR DESCRIPTION
**Pull request by sarahwooddit**
To do (delete all that do not apply):

 - [ ] Change has a jira ticket that has the correct status.
 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [ ] (if adding CSS) CSS have been compiled.
 - [ ] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags
 - [ ] (if changing the UI) Add a printscreen to the PR

### CODE REVIEW

**Comment by sarahwooddit** [comment]
Changing the complete screens for individuals was a bit more complicated than expected. Had to change view.py. I've also guessed at the url for events. Sorry!

File: enrolment/templates/enrolment/individual-interstitial.html [hunk 1]
```diff
@@ -11,20 +11,29 @@
       {% endwith %}
```
Comment (richtier) [inline enrolment/templates/enrolment/individual-interstitial.html]: this is what the resulting page looks like

image

is that as expected?

Comment (richtier) [reply enrolment/templates/enrolment/individual-interstitial.html]: I removed the duplicate "go back" and added some spacing

image

@sarahwooddit you dig?

Comment (sarahwooddit) [reply enrolment/templates/enrolment/individual-interstitial.html]: Hey Rich, the continue for individual link needs to go first, before the return to great link. Could you swap them. Looks much better with your spacing. Also, the back to select your business type link looks very close to the head line. Is there meant to be some spacing there?

Comment (richtier) [reply enrolment/templates/enrolment/individual-interstitial.html]: I'll show the print screen to CB to understand if there should be more spacing. sure will move the contine above

Comment (richtier) [reply enrolment/templates/enrolment/individual-interstitial.html]: newest screen
image

Comment (richtier) [reply enrolment/templates/enrolment/individual-interstitial.html]: the "Back to Select your..."  - is that mean to be capitalised S on select?

Comment (sarahwooddit) [reply enrolment/templates/enrolment/individual-interstitial.html]: That's what it had in the design - https://projects.invisionapp.com/share/M7SKLCR2FN3#/screens/369418907

Is that how we usually do it?

**Pull request merged**